ddtrace/profiling/collector/threading.py

```diff
@@ -71,7 +82,10 @@ def __init__(self, wrapped, recorder, tracer, max_nframes, capture_sampler, endp
         self._self_endpoint_collection_enabled = endpoint_collection_enabled
         frame = sys._getframe(2 if WRAPT_C_EXT else 3)
         code = frame.f_code
-        self._self_name = "%s:%d" % (os.path.basename(code.co_filename), frame.f_lineno)
+        self._self_name = "%s:%d" % (
```

Comment:
how is this name used? would `%r` be more useful? (my guess is "no")

Reply:
It's shown as the lock name in the UI.

(This is unrelated to the PR though, there's no change here)